src/BoundaryConditions/DirichletBoundaryConditions.jl

```diff
@@ -3,19 +3,20 @@ Module to handle Dirichlet boundary conditions.
 """
 module DirichletBoundaryConditions
 
-using ..Utils, ..BoundaryConditions
```

Comment:
We might add this kind of format preferences into the CONTRIBUTING.md ?

Reply:
yes, or just have a template with these settings

Reply:
Oh great, is already done. Merging then... 👍🏼